src/components/sensitivity-analysis-result-tab.js

```diff
@@ -165,7 +183,13 @@ function PagedSensitivityResult({
         next.isFetchNeedy = prev.isFetchNeedy;
         next.askedFilterVersion = prev.askedFilterVersion;
     } else if (!prev.fetched) {
-        // probably fetch failed, worth retrying
+        // probably fetch failed, worth retrying ?
+        if (retryCount >= RETRY_LIMIT_COUNT) {
```

Comment:
You don't need this retryCount. 
When there's no result, the behavior should be the same as when there's some results and should not refetch the result.
Removing setAllowsShowingLoader(false) can help

Reply:
It will not re-fetch if the HTTP 204 is well handled beneath, as it s now Tang has corrected the fetch in rest-api.js.
The case it would try some more times is when a 4xx or 5xx is received.
The setAllowsShowingLoader(false) at row 264 is the point of the PR : prevent the loader to show too early.

Reply:
Yes but it's prevented as allowShowingLoader is initiated to false

Reply:
The first time the table show : yes. Next times, when filtering/sorting/paging parameters change, it has to be set back to false in order to not have the loader appears too early.